common/middleware/storeOasysSessionKey.js

```diff
@@ -0,0 +1,9 @@
+// adds OASys session key to session. This should then be passed as header X-Session-Id in each API call
+
+module.exports = (req, res, next) => {
+  const { sessionKey } = req.query
+  if (sessionKey) {
+    req.session.sessionKey = sessionKey
```

Comment:
Code style thing. Could just be a oneliner? Your call?
`if (sessionKey) req.session.sessionKey = sessionKey`

Reply:
Although..... is its omission an error scenario? Shouldn't it always be there?

Reply:
I'm not a fan of one liners (no prob if other people use them though) - I like the safety of the braces! 

Discussed with Stuart when I started this story - we're not assuming there will always be a sessionKey on the query string so no error necessary, just store/update it when it is there and ignore when it isn't.